package.json

```diff
@@ -48,17 +48,18 @@
   },
   "homepage": "https://github.com/ng-bootstrap/ng-bootstrap#readme",
   "dependencies": {
-    "@angular/core": "4.0.3",
-    "@angular/common": "4.0.3",
-    "@angular/forms": "4.0.3"
+    "@angular/animations": "4.2.0",
```

Comment:
Do we need to force people to have `@angular/animations` included? I mean, could the `@angular/animations` module be optional (and in this case we would fall back to the current state without animations? Or would it fail miserably?

Reply:
It would fail due to the new imports from the `@angular/animations` package. Folks that don't want animations can just add the `NoopAnimations` module though. From the users point of view it's a case of `npm install @angular/animations` and adding the module to their root AppModule, so an acceptable breaking change imo.

Reply:
Yes, breaking change is fine. I was more worried about payload increase for folks not wanting / needing animations.

Reply:
Looking at https://unpkg.com/@angular/animations@4.3.6/bundles/ the gzipped size of `animations.umd.min.js` is 1.3kb and `animations-browser.umd.min.js` is 22.9kb. I would expect that to drop though with proper tree shaking 🤞